### PR TITLE
feat(zone): added presets for goodstore data

### PR DESCRIPTION
**Pull request by lidiyacommercetools**
Adding zone presets to the good store data. Based on the presets used for the sample fashion data. 

Data to be mirrored: 
https://github.com/frontastic-developers/customer-thegoodstore/blob/master/getting-started/data/zones.json

### CODE REVIEW

File: models/zone/src/zone/zone-draft/presets/sample-data-goodstore/country-germany.spec.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { TZoneDraft, TZoneDraftGraphql } from '../../../types';
+import countryGermany from './country-germany;
```
Comment (stephsprinkle) [inline models/zone/src/zone/zone-draft/presets/sample-data-goodstore/country-germany.spec.ts:2]: You're missing an end quote!

```suggestion
import countryGermany from './country-germany';
```

**Review by stephsprinkle — COMMENTED** [review]
Amazing first PR!!

File: models/zone/src/zone/zone-draft/presets/sample-data-goodstore/country-greatbritain.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import * as ZoneDraft from '../../index';
+
+const countryGreatBritain = () =>
+  ZoneDraft.presets
+    .empty()
+    .name('Great Britain')
```
[the remark this answers is not in the document]
Comment (lidiyacommercetools) [reply models/zone/src/zone/zone-draft/presets/sample-data-goodstore/country-greatbritain.ts]: @stephsprinkle , updated to UK

File: models/zone/src/zone/zone-draft/presets/sample-data-goodstore/country-greatbritain.spec.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import countryGreatBritain from './country-greatbritain';
+
+describe('with the preset `country great britain`', () => {
+  it('should return a zone with name `United Kingdom`', () => {
+    const zone = countryGreatBritain().build<TZoneDraft>();
+
+    expect(zone).toMatchInlineSnapshot(`
+      {
+        "description": undefined,
+        "key": "greatbritain",
```
[the remark this answers is not in the document]
Comment (lidiyacommercetools) [reply models/zone/src/zone/zone-draft/presets/sample-data-goodstore/country-greatbritain.spec.ts]: Updated

File: models/zone/src/zone/zone-draft/presets/sample-data-goodstore/index.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import countryGermany from './country-germany';
+import countryGreatBritain from './country-unitedkingdom';
+
+const presets = {
+  countryGermany,
+  countryGreatBritain,
```
Comment (stephsprinkle) [inline models/zone/src/zone/zone-draft/presets/sample-data-goodstore/index.ts:6]: One last modification 😅

```suggestion
import countryUnitedKingdom from './country-unitedkingdom';

const presets = {
  countryGermany,
  countryUnitedKingdom,
```

**Review by stephsprinkle — APPROVED** [review]
🎉🎉🎉

File: models/zone/src/zone/zone-draft/presets/sample-data-goodstore/country-germany.spec.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+        "locations": [
+          {
+            "country": "DE",
+          },
+        ],
+        "name": "Europe",
+      }
+    `);
+  });
+});
```
Comment (jaikamat) [inline models/zone/src/zone/zone-draft/presets/sample-data-goodstore/country-germany.spec.ts:38]: Nit: Formatting issues (files need a newline), but that's a small tweak

**Comment by jaikamat** [comment]
@lidiyacommercetools A PM WHO CODES? We're the luckiest team on planet earth 🎊

**Review by jaikamat — APPROVED** [review]

**Comment by tylermorrisford** [comment]
Looks like you just need a couple newlines at the ends of files and to update the inline snapshots in your tests, but this is straight 🏆 #winning.

File: .changeset/eight-birds-invite.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (stephsprinkle) [inline .changeset/eight-birds-invite.md:1]: It looks like an extra changeset snuck into this PR

**Pull request closed without merging**